tools/quadrados/src/import.cpp

```diff
@@ -0,0 +1,89 @@
+#include <filesystem>
+#include <fstream>
+#include <iostream>
+#include <random>
+#include <sstream>
+
+#include "tools.hpp"
+
+namespace fs = std::filesystem;
+
+/// Generates a UUID-like random string (with hyphens),
+/// consisting of digits [0-9] and lowercase letters [a-z].
+/// @returns A 36-character string in the format xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+static std::string generateRandomStringUUID()
+{
+    std::random_device rd;
+    std::mt19937 gen(rd());
+    const char charset[] = "0123456789"
+                           "abcdefghijklmnopqrstuvwxyz";
```

Comment:
You don't need to do all this!
Doing just `auto id = uuids::uuid_random_generator(gen)();` is enough.